subworkflows/local/taxonomy/main.nf

```diff
@@ -6,11 +6,17 @@
 | MODULES AND SUBWORKFLOWS |
 ***************************/
 
-include { BBMERGE } from "../../../modules/local/bbmerge"
-include { SUMMARIZE_BBMERGE } from "../../../modules/local/summarizeBBMerge"
+if (params.single_end) {
```

Comment:
I don't think this "if" statement here is helpful, I would remove it especially since you use the if statement to differentiate the runs in the main code.

Reply:
fixed

Reply:
@willbradshaw for you to discuss with @harmonbhasin  how to handle processes that are only used by either a single-end run or a paired-end run.

Reply:
@simonleandergrimm i chatted with Will, he said that it's okay to remove the if statement (we only previously had the if statement because nextflow would throw warnings, but that's been removed now). Please update this file to reflect this (i.e. remove the if statement).